paasta_tools/utils.py

```diff
@@ -2493,6 +2493,7 @@ def get_git_config(self) -> Dict:
                     "yelpsoa-configs": {
                         "repo_name": "yelpsoa-configs",
                         "git_server": DEFAULT_SOA_CONFIGS_GIT_URL,
+                        "primary_server": DEFAULT_SOA_CONFIGS_GIT_URL,
```

Comment:
nit: I am not sure the name `primary` here is as evocative as it could be (I have all the context and it took me a sec).  Might be better to call it something like `trigger_deploys_on_server` or something like that.

Reply:
I changed it to `deploy_server` instead of `primary_server`.